weeather/LoadBots.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Text.Json;
+using weather.Entities;
+
+namespace weather
+{
+    internal class LoadBots
+    {
+
+        public List<Bot> LoadBotsFromConfigFile()
+        {
+            string data = ConfigurationFileData();
+            return ParseToBots(data);
+        }
+
+        private string ConfigurationFileData()
```

Comment:
let's rename the method `loadConfigFile`

Reply:
Can't agree more